ui/src/alerting/components/CheckHistory.tsx

```diff
@@ -0,0 +1,113 @@
+// Libraries
+import React, {useMemo, FC} from 'react'
+import {Page} from '@influxdata/clockface'
+import {connect} from 'react-redux'
+
+// Components
+import {ResourceIDsContext} from 'src/alerting/components/AlertHistoryIndex'
+import EventViewer from 'src/eventViewer/components/EventViewer'
+// import CheckHistoryControls from 'src/alerting/components/CheckHistoryControls'
```

Comment:
missed comment!

Reply:
wait- why isn't this being imported? this is the search bar I think it should be in the page? If you don't intend to have the search bar showing let's not create the page as a part of this PR.

Reply:
I reinstated the search bar, i think it was commented out from when we discussed if it worked